test/unit/plugin/v2/installer_test.rb

```diff
@@ -0,0 +1,6 @@
+# it should be able to list installed plugins
```

Comment:
Do we need this file?

Reply:
It was just my way of keep notes prior to a branching strategy bing determined. I'll remove them, as these (implemented) tests will come in on another branch.